setup-cf-kube-dist.sh

```diff
@@ -1,9 +1,19 @@
 #!/bin/bash
 
-target="$1"
+set -o nounset
+set -o errexit
 
-# EV 'PIPELINE_PREFIX' = prefix to pipeline name for local
-#                        customization of test pipelines
+# EV 'PIPELINE_PREFIX' = prefix to pipeline name for local customization of test
+#           pipelines
+
+PIPELINE_NAME=cf-kube-dist
+
+if test -z "${1:-}"; then
+    printf "Usage:\n%s <target>\n" "${0}" >&2
+    exit 1
+fi
+
+target="${1}"
```

Comment:
I've updated this to follow what the helm deploy script is doing: if it's not set, use `fly`'s default. If you want to override it, set the target.